pkg/releasesync/releasesync.go

```diff
@@ -43,27 +49,35 @@ func (r *releaseSyncOptions) Run() error {
 
 		for _, tags := range allTags {
 			for _, tag := range tags.Tags {
-				c := int64(0)
-				r.db.DB.Table("release_tags").Where(`"release_tag" = ?`, tag.Name).Count(&c)
-				if c > 0 {
+				mReleaseTag := models.ReleaseTag{}
+				r.db.DB.Table(release_tags_table).Where(`"release_tag" = ?`, tag.Name).Find(&mReleaseTag)
+				// expect Phase to be populated if the record is present
+				if len(mReleaseTag.Phase) > 0 {
+
+					if mReleaseTag.Phase != tag.Phase {
+						log.Infof("Fetching updated tag %s from release controller...\n", tag.Name)
+						releaseTag := r.buildReleaseTag(tags.Architecture, release, tag)
```

Comment:
Do we need to fetch it again in this case?


```go
if mReleaseTag.Phase != tag.Phase {
  mReleaseTag.Phase = tag.Phase
  mReleaseTag.Forced = true
  if err := r.db.DB.Table(release_tags_table).Save(mReleaseTag).Error; err != nil {
  [...]
```

Reply:
Although your version is safer, in case anything else changes.

Reply:
I think I did it that way to force it to reprocess and set the forced flag via the api.  But if performance is a concern we could definitely do it this way.  We could even just update the individual columns.  I started to play with that but seemed like we would have to code in the column names or go through some funky reflection to get it out of gorm tag.

Reply:
We  only get a handful of releases a day, performance isn't too much of a concern.  Your version is good